app/components/admin/groups/group_form_component.html.erb

```diff
@@ -1,50 +1,39 @@
+        <p class="self-stretch flex-grow w-[521px] text-xl font-semibold text-left text-gray-900"><%= @group.name %></p>
+        <p class="flex-grow-0 flex-shrink-0 text-xs text-left text-gray-500">Používatelia v skupine</p>
+      </div>
+    </div>
+    <div class="flex flex-col justify-start items-start self-stretch flex-grow-0 flex-shrink-0">
+      <% if !@readonly %>
+        <%= form_with url: search_non_members_admin_tenant_group_path(Current.tenant, @group),
                           data: { controller: 'debounce', debounce_target: 'form'},
                           class: "flex justify-start items-center self-stretch flex-grow-0 flex-shrink-0 gap-4 p-6 border-t-0 border-r-0 border-b border-l-0 border-gray-200" do |form| %>
-              <div class="flex w-full items-center rounded-md shadow-sm ring-1 ring-inset ring-gray-300 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-600">
-                <div class="p-2">
-                  <%= render Icons::MagnifyingGlassComponent.gray(size: "5") %>
-                </div>
-                <%= tag.turbo_frame id:'user-search-results' %>
-                <%= form.search_field :name_search, value: params[:name_search], placeholder: "Vyhľadaj používateľa pre pridanie do skupiny",
+          <div class="flex w-full items-center rounded-md shadow-sm ring-1 ring-inset ring-gray-300 focus-within:ring-2 focus-within:ring-inset focus-within:ring-indigo-600">
+            <div class="p-2">
+              <%= render Icons::MagnifyingGlassComponent.gray(size: "5") %>
+            </div>
+            <%= tag.turbo_frame id:'user-search-results' %>
+            <%= form.search_field :name_search, value: params[:name_search], placeholder: "Vyhľadaj používateľa pre pridanie do skupiny",
                   data: { action: 'input->debounce#debounce' },
                   onreset: "this.form.requestSubmit()",
                   class: "block w-full flex-1 border-0 bg-transparent py-1.5 pl-1 text-gray-900 placeholder:text-gray-400 focus:ring-0 sm:text-sm sm:leading-6" %>
-              </div>
-            <% end %>
-          <% end %>
-          <div class="flex flex-col justify-start items-start self-stretch divide-y relative gap-2 p-2 border-t-0 border-r-0 border-b border-l-0 border-gray-200">
-            <% @group.group_memberships.each do |group_membership| %>
-              <div class="self-stretch p-6 flex justify-start items-start gap-6">
-                <% if !@readonly %>
-                  <%= render Admin::Groups::MembersListRowComponent.new(group_membership) %>
-                <% else %>
-                  <%= render Admin::Groups::MembersListRowViewOnlyComponent.new(group_membership) %>
-                <% end %>
-              </div>
+          </div>
+        <% end %>
+      <% end %>
+      <div class="flex flex-col justify-start items-start self-stretch divide-y relative gap-2 p-2 border-t-0 border-r-0 border-b border-l-0 border-gray-200">
```

Comment:
Funguje vsetko ok. Jedina vec, co sa mi trochu nepaci je vzhlad, ked nikto nie je pridany do skupiny. Tie dve ciary dole pod sebou.
<img width="1381" alt="image" src="https://github.com/user-attachments/assets/7c190c61-760c-4459-81b3-d8eba8a2102f" />
To ale tak bolo aj doteraz. Dalo by sa vyriesit jednoducho, zabalenim tohto div do if: `<% if @group.group_memberships.any? %>`

Reply:
nejaky blank stav by sa tu hodil. Uz ich mame aj inde cize staci asi skopcit a upravit.

Reply:
@richardlences pozries prosim este toto?